code/game/objects/structures/barricade/deployable.dm

```diff
@@ -133,6 +115,8 @@
 	)
 	icon = 'icons/obj/items/marine-items.dmi'
 
+	var/list/stack_health = list(350)
```

Comment:
use initial to get the default health here, not hardcoded. you’ll probably need to do it in init